crates/astria-conductor/src/bin/astria-conductor.rs

```diff
@@ -35,25 +27,24 @@ use tracing::{
     instrument,
 };
 
+const EXIT_CONFIG_CODE: i32 = 78;
+
 #[tokio::main]
 async fn main() -> Result<()> {
-    run().await?;
+    if let Err(e) = run().await {
+        eprintln!("Exited with error: {e}");
+        // FIXME: might have to bubble up exit codes, since we might need 
+        //        to exit with other exit codes if something else fails
+        std::process::exit(EXIT_CONFIG_CODE);
```

Comment:
Yeah, looks like thisis ok for now but will need to be fixed. Can you create an issue to track this and provide the url of the issue in the `FIXME(https://github.com/...)` above?